exporters/committime/collector_base.py

```diff
@@ -273,3 +285,25 @@ def _get_repo_from_build_config(self, build):
                     return git_uri + ".git"
 
         return None
+
+    def _get_revision_from_build(self, build):
+        """
+        Determines the git revision from the parent BuildConfig
+        :param build: the Build resource
+        :return: revisions as a str or None if not found
+        """
+        if build.metadata.annotations.buildSpecSourceGitUri:
```

Comment:
Is this mistake? Should it be:

`if build.metadata.annotations.buildSpecRevisionGitCommit:`

I also don't understand the logic:

```
if something:
   return something
return None
```

This is equivalent to just:
`return something`

Because:

```
something = None
if something:
   return something
return None
```

Unless you wanted to return None if string is empty, then check the len(something) as well.

But as on lines 276-277, I would add some validaitons if the revision is valid regex.